package.json

```diff
@@ -39,7 +39,7 @@
     "os-locale": "2.1.0",
     "progress": "^1.1.8",
     "qrcode-terminal": "^0.11.0",
-    "request": "2.82.0",
+    "request": "2.88.0",
```

Comment:
is the `yargs` dependency also to be updated in this PR?

Reply:
Yes, the yargs was updated. It was defined in the pkg json as 12.0.1 already but not in the lock file. You can see the commit regards this specific actions. Also, as both are minor versions than it can be made in the same PR as defined previously.